.env.integration

```diff
@@ -5,3 +5,4 @@ COMPRESSION_ENABLED=False
 AWS_REGION=eu-west-2
 DJANGO_LOG_LEVEL=WARNING
 OPENAI_MODEL=azure/gpt-35-turbo-16k
+ROOT_INDEX=redbox-data-integration
```

Comment:
Could we rename this to `ELASTIC_ROOT_INDEX` to be more descriptive, please?

Reply:
[7c6003e](https://github.com/i-dot-ai/redbox-copilot/pull/482/commits/7c6003e777f0ad6915c13838b5db12b5e4cb3acc)